examples/next-js-simple/pages/[[...page]].tsx

```diff
@@ -39,7 +39,7 @@ export async function getStaticPaths() {
 
   return {
     paths: pages.map((page) => `${page.data?.url}`),
-    fallback: true,
+    fallback: 'blocking',
```

Comment:
Would't this 404 on new urls in draft content? ( prevents editing and previewing draft pages )

Reply:
Hasn’t been an issue that I’ve seen anywhere, the only behavior change is not flashing a janky blank loading page that makes it appear broken